libvcx/src/aries/handlers/issuance/issuer/state_machine.rs

```diff
@@ -179,16 +179,16 @@ impl IssuerSM {
         None
     }
 
-    pub fn state(&self) -> u32 {
+    pub fn get_state(&self) -> IssuerState {
```

Comment:
I see also couple changes here which projects into breaking changes on numeric level - please include in PR description. Not sure if it's worthy to go into details if which state number changed exactly how, but at least note that state mapping has changed for this FSM.